CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Release notes
 This page documents production updates to Cloud Tools for IntelliJ. You can check this page for announcements about new or updated features, bug fixes, known issues, and deprecated functionality.
 
+## 18.4.1
+
+### Added
+  - Let the Google Cloud Tools plugin manage the Cloud SDK installation for you. No longer any need to manually download the SDK. [673](https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/673)
```

Comment:
Could we also add `installation and update` here to mention update will also be handled?